Sources/SwiftParser/Lexer/Cursor.swift

```diff
@@ -711,14 +713,17 @@ extension Lexer.Cursor {
         }
 
       case UInt8(ascii: "\n"), UInt8(ascii: "\r"):
-        isMultiline = true
+        newlinePresence = .present
         continue
       case nil:
-        return isMultiline
+        error = LexingDiagnostic(.unterminatedBlockComment, position: afterStar)
```

Comment:
Any reason for putting this at the character after the `*`? I believe we usually place unterminated diagnostics either on the opening character (i.e the opening `/` in this case), or at the end of where we finished lexing. The C++ parser appears to put it at the end, with a note at the start. Ideally I guess we'd do something similar, but until we support notes for lexing diagnostics, maybe we should just put it at the end for now?

Reply:
Because I was too lazy to pass the cursor pointing to the slash into the function. Changed it now